src/main/java/mil/dds/anet/database/LocationDao.java

```diff
@@ -59,9 +59,9 @@ public Location insertInternal(Location l) {
     final Handle handle = getDbHandle();
     try {
       handle.createUpdate(
-          "/* locationInsert */ INSERT INTO locations (uuid, name, type, description, status, lat, lng, digram, trigram, "
+          "/* locationInsert */ INSERT INTO locations (uuid, name, type, description, status, lat, lng, \"geoJson\", digram, trigram, "
               + "\"createdAt\", \"updatedAt\", \"customFields\") VALUES (:uuid, :name, :type, :description, :status, "
-              + ":lat, :lng, :digram, :trigram, :createdAt, :updatedAt, :customFields)")
+              + ":lat, :lng, :geoJson, :digram, :trigram, :createdAt, :updatedAt, :customFields)")
```

Comment:
I believe we agreed that the `geoJson` field should effectively be read-only, so it should not appear in this INSERT.